sql/core/src/test/scala/org/apache/spark/sql/errors/QueryExecutionErrorsSuite.scala

```diff
@@ -875,6 +875,18 @@ class QueryExecutionErrorsSuite
       sqlState = "XX000")
   }
 
+  test("INTERNAL_ERROR: Calling eval on Unevaluable NamedArgumentExpression") {
```

Comment:
I don't think we need to test it. `Unevaluable` is not added by this PR and has been there for a long time.